charts/langsmith/templates/frontend/config-map.yaml

```diff
@@ -39,6 +39,15 @@ data:
                 return 200;
          }
 
+        location ~ ^/v1/platform/ {
```

Comment:
i will do this when i bump the next release so you can ignore for now(will need the new images after release)

Reply:
so no need to merge this right now?

Reply:
nope, we peg to a specific version